multichain-testing/test/auto-stake-it.test.ts

```diff
@@ -18,15 +18,19 @@ const accounts = ['agoricAdmin', 'cosmoshub', 'osmosis'];
 
 const contractName = 'autoAutoStakeIt';
 const contractBuilder =
-  '../packages/builders/scripts/testing/start-auto-stake-it.js';
+  '../packages/builders/scripts/testing/init-auto-stake-it.js';
 
 test.before(async t => {
-  const { deleteTestKeys, setupTestKeys, ...rest } = await commonSetup(t);
+  const { setupTestKeys, ...common } = await commonSetup(t);
+  const { assetInfo, chainInfo, deleteTestKeys, startContract } = common;
   deleteTestKeys(accounts).catch();
   const wallets = await setupTestKeys(accounts);
-  t.context = { ...rest, wallets, deleteTestKeys };
-  const { startContract } = rest;
-  await startContract(contractName, contractBuilder);
+  t.context = { ...common, wallets };
+
+  await startContract(contractName, contractBuilder, {
+    chainInfo: JSON.stringify(chainInfo),
```

Comment:
why must every call stringify? please make`startContract` do that itself. It's specific to multichain-testing so it can know about these two properties even if `deployBuilder` doesn't. (e.g. if there is an intent to use that module or something like it in more places)

Reply:
Done